DeepSpeech.py

```diff
@@ -468,7 +468,8 @@ def train():
     }
 
     # Building the graph
-    optimizer = create_optimizer()
+    learning_rate_tensor = tf.placeholder(tf.float32, shape=[])
```

Comment:
Will this break the old checkpoints?

Reply:
Yes.

Reply:
Can we/you do this in another pull request then? Maybe shortly before you release a new model.

I need your released english checkpoint because im using it as base for my training in german.

Reply:
We can also just hold this PR until shortly before we release a model, if you need that. One thing I'll note is that you could write some small amount of code to handle this case, by initializing the learning rate variable if it's not found in the checkpoint. But this will introduce conflicts with the transfer learning PR so I'd just wait.

Reply:
I have to admit i'd love to see that in 0.7.0, both because I'd want to experiment with it, and because that would give us more time, if it can't be landed in a compatible way, then let's wait for after 0.7/

Reply:
We can fix the 0.7 checkpoint to have the variable in it as well, it should be quite simple.

Reply:
Wont saving the learning rate in a variable make transfer learning more complicated as you have to modifiy the graph to change the learning rate back to a higher start value again?

Reply:
You don't have to modify the graph, just assign a value to the variable. Transfer learning already requires extensive logic around checkpoint loading anyway so this would not be a complex feature.

Reply:
Saving the learning rate in a variable now